src/PriceCalculatorKata/Product.cs

```diff
@@ -2,6 +2,11 @@
 
 namespace PriceCalculatorKata
 {
+    enum DiscountPrecedence
+    {
+        BeforeTax,
```

Comment:
Better assign a specific integer to each value here like so:
BeforeTax = 0,
AfterTax = 1

I know there's a default value/integer for each already, but if someone else tries to add another enum value, and try to add it to the beginning (like before BeforeTax) then this new value will have zero. Which gonna missed up the old values.

This matter if you save this DiscountPrecedence for example in DB. Like if you have a column in Product table of type INT to hold the discountPrecedence.